dropwizard-testing/src/test/java/io/dropwizard/testing/app/ResourceTestWithGrizzly.java

```diff
@@ -19,13 +20,32 @@
     @ClassRule
     public static final ResourceTestRule resources = ResourceTestRule.builder()
             .addResource(new ContextInjectionResource())
-            .setTestContainerFactory(new GrizzlyTestContainerFactory())
+            .setTestContainerFactory(new GrizzlyWebTestContainerFactory())
+            .addProvider(new RuntimeExceptionMapper())
             .build();
 
+    private static class RuntimeExceptionMapper implements ExceptionMapper<RuntimeException> {
```

Comment:
I'm nitpicking again, but it would be great if you moved this class to the start or to the end of `ResourceTestWithGrizzly`. It makes the outer class more focused.